test/test_macros.rb

```diff
@@ -126,4 +127,22 @@ def render(opts)
     @wiki.write_page("ListNamedArgsPage", :markdown, "<<ListNamedArgs(xyzzy=\"Foo\")>>", commit_details)
     assert_match(/@xyzzy = Foo@/, @wiki.pages[0].formatted_data)
   end
+
+  test "Video macro given a valid mp4 file displays an html5 video tag with a video type of mp4" do
+    file = "/assets/vid.mp4"
```

Comment:
Rather than committing the test video files to the repo, it may be cleaner to use [stubs](https://github.com/freerange/mocha) here: `File.stubs(:exists?).with(testpath('examples', file)).returns(true)`. Also see the above comment about using `File.exists?`. But again: it would be even better if we don't need to check for file existence, and can just leave that to the browser.